lib/redux/actions/cash_wallet_actions.dart

```diff
@@ -314,16 +325,15 @@ ThunkAction joinCommunityCall({dynamic community, dynamic token}) {
   };
 }
 
-ThunkAction switchCommunityCall({String communityAddress}) {
+ThunkAction switchCommunityCall(String communityAddress) {
   return (Store store) async {
     try {
-      communityAddress = communityAddress ?? Web3.getDefaultCommunity();
       store.dispatch(new SwitchCommunityRequested(communityAddress));
       dynamic community =
-          await graph.getCommunityByAddress(communityAddress: communityAddress);
+        await graph.getCommunityByAddress(communityAddress);
       logger.i('community fetched for $communityAddress');
       dynamic token =
-          await graph.getTokenOfCommunity(communityAddress: communityAddress);
+          await graph.getTokenOfCommunity(communityAddress);
       logger.i('token ${token["address"]} fetched for $communityAddress');
       store.dispatch(joinCommunityCall(community: community, token: token));
       return store.dispatch(new SwitchCommunitySuccess(
```

Comment:
are you network aware? what about server? are you changing the server host in the switch community?

Reply:
It uses graphql and the new v2 API, so it doesn't care if the token originated on ropsten or mainnet.  It's uses the same graph and the same server for both.

Reply:
We should handle qa/prod environment, but I think it's not related to this task